### PR TITLE
[skip-ci] Update README with API section

### PR DESCRIPTION
**Pull request by Vijar**
@redonkulus

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by jedireza** [comment]
:ship:

**Comment by redonkulus** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -31,24 +31,34 @@ var pluginInstance = routrPlugin({
...
-### actionContext
+#### Constructor(options)
 
-Provides full access to the routr instance. See [routr docs](https://github.com/yahoo/routr) for more information.
+Creates a new fetchr plugin instance with the following parameters:
 
- * `actionContext.router.makePath(routeName, routeParams)`: Create a URL based on route name and params
- * `actionContext.router.getRoute(path)`: Returns matched route
+ * `options`: An object containing the plugin settings
+ * `options.routes` (optional): Stores your routes configuration
```
Comment (Vijar) [inline README.md]: required

File: README.md [hunk 2]
```diff
@@ -31,24 +31,34 @@ var pluginInstance = routrPlugin({
...
 app.plug(pluginInstance);
 ```
+[//]: # (API_START)
+## Routr Plugin API
 
-## Fluxible Methods Added
+### Instance Methods
 
-### actionContext
+#### Constructor(options)
```
Comment (Vijar) [inline README.md]: not instance method

**Pull request merged**